managed/CounterStrikeSharp.API/Modules/Memory/VirtualFunctions.cs

```diff
@@ -68,7 +67,13 @@ public static class VirtualFunctions
     
     public static MemoryFunctionWithReturn<CCSPlayer_WeaponServices, CBasePlayerWeapon, bool> CCSPlayer_WeaponServices_CanUseFunc = new(GameData.GetSignature("CCSPlayer_WeaponServices_CanUse"));
     public static Func<CCSPlayer_WeaponServices, CBasePlayerWeapon, bool> CCSPlayer_WeaponServices_CanUse = CCSPlayer_WeaponServices_CanUseFunc.Invoke;
-    
+
+    public static MemoryFunctionWithReturn<int, string, CCSWeaponBaseVData> GetCSWeaponDataFromKeyFunc = new(GameData.GetSignature("GetCSWeaponDataFromKey"));
+    public static Func<int, string, CCSWeaponBaseVData> GetCSWeaponDataFromKey = GetCSWeaponDataFromKeyFunc.Invoke;
+
+    public static MemoryFunctionWithReturn<CCSPlayer_ItemServices, CEconItemView, AcquireMethod, NativeObject, AcquireResult> CCSPlayer_CanAcquireFunc = new(GameData.GetSignature("CCSPlayer_CanAcquire"));
+    public static Func<CCSPlayer_ItemServices, CEconItemView, AcquireMethod, NativeObject, AcquireResult> CCSPlayer_CanAcquire = CCSPlayer_CanAcquireFunc.Invoke;
+
```

Comment:
LGTM, however I wonder if would it be possible to wrap extension methods around these? so we would not have to do `VirtualFunctions.CCSPlayer_CanAcquireFunc(..)` but `itemServices.CanAcquire(..)`

also use proper gamedata names, please rename from `CCSPlayer_CanAcquire` to `CCSPlayer_ItemServices_CanAcquire` (same with the other one if you know the exact class name)